generators/php/sdk/src/endpoint/http/HttpEndpointGenerator.ts

```diff
@@ -329,27 +332,6 @@ export class HttpEndpointGenerator extends AbstractEndpointGenerator {
         });
     }
 
-    private throwNewBaseException({ message }: { message: php.CodeBlock }): php.CodeBlock {
-        return php.codeblock((writer) => {
-            writer.write("throw ");
-            writer.writeNode(
-                php.instantiateClass({
-                    classReference: this.context.getBaseExceptionClassReference(),
-                    arguments_: [
-                        {
-                            name: "message",
-                            assignment: message
-                        },
-                        {
-                            name: "previous",
-                            assignment: php.codeblock("$e")
-                        }
-                    ]
-                })
-            );
-        });
-    }
-
```

Comment:
Moved this out to a [utility module](https://github.com/fern-api/fern/pull/4789/files#diff-c38a4e8988108dbf790294bf084b4dea9e7227215b9ab3026a64b27e0c69032f) so I could reuse in another class.